### PR TITLE
Fix article has no title

### PR DESCRIPTION
**Pull request by MengZn**
The title of this article is not displayed on this page, I found this error and fixed it.

### CODE REVIEW

File: content/en/docs/setup/release/notes.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+---
+title: Release Notes
+content_template: templates/concept
+---
 <!-- BEGIN MUNGE: GENERATED_TOC -->
```
Comment (stewart-yu) [inline content/en/docs/setup/release/notes.md]: change like this
```
---
title: Release Notes
content_template: templates/concept
weight: 40
---

{{% capture overview %}}

```

File: content/en/docs/setup/release/notes.md [hunk 2]
```diff
@@ -843,4 +847,3 @@ filename | sha256 hash
 * Fix kubectl drain --timeout option when eviction is used. ([#64378](https://github.com/kubernetes/kubernetes/pull/64378), [@wrdls](https://github.com/wrdls))
 * This PR will leverage subtests on the existing table tests for the scheduler units. ([#63659](https://github.com/kubernetes/kubernetes/pull/63659), [@xchapter7x](https://github.com/xchapter7x))
     * Some refactoring of error/status messages and functions to align with new approach.
-
```
Comment (stewart-yu) [inline content/en/docs/setup/release/notes.md]: keep origin

File: content/en/docs/setup/release/notes.md (hunk 1 above)
Comment (MengZn) [reply content/en/docs/setup/release/notes.md]: Thanks already fix it.

File: content/en/docs/setup/release/notes.md [hunk 3]
```diff
@@ -1,4 +1,15 @@
+---
+title: Release Notes
+content_template: templates/concept
+---
+{{% capture overview %}}
+
+[Documentation](https://docs.k8s.io) & [Examples](https://releases.k8s.io/release-1.12/examples)
+
```
Comment (stewart-yu) [inline content/en/docs/setup/release/notes.md]: Why add this line?

**Comment by stewart-yu** [comment]
After your change, please check the link https://deploy-preview-10670--kubernetes-io-master-staging.netlify.com/docs/setup/release/#downloads-for-v1-12-0-rc-1.  Thanks

File: content/en/docs/setup/release/notes.md (hunk 3 above)
Comment (MengZn) [reply content/en/docs/setup/release/notes.md]: In this commit #10258 ,this line still exists.
In next commit #10292 ,it It disappeared.

File: content/en/docs/setup/release/notes.md (hunk 2 above)
Comment (MengZn) [reply content/en/docs/setup/release/notes.md]: resolve it,thank .

**Comment by MengZn** [comment]
@stewart-yu can you help me review it again please

**Comment by stewart-yu** [comment]
image
apppear in many places in docs, that's normally?
can you check the preview link after changed?

**Comment by MengZn** [comment]
> image
> apppear in many places in docs, that's normally?
> can you check the preview link after changed?


Thanks for your suggestion, the recurring part has been removed.

File: content/en/docs/setup/release/notes.md [hunk 4]
```diff
@@ -259,7 +225,7 @@ filename | sha256 hash
 
 # v1.12.0-beta.2
 
-[Documentation](https://docs.k8s.io) & [Examples](https://releases.k8s.io/release-1.12/examples)
+
```
Comment (stewart-yu) [inline content/en/docs/setup/release/notes.md]: Revert all fixed in this PR please, all fixed in https://github.com/kubernetes/website/pull/10692. Could you like to help fix the `Example` link?

Comment (jimangel) [reply content/en/docs/setup/release/notes.md]: @stewart-yu @MengZn where should examples link to? Should we just remove it?

Comment (MengZn) [reply content/en/docs/setup/release/notes.md]: @jimangel Is it better to reconnect Example to this https://github.com/kubernetes/website/tree/release-1.12/content/en/examples?

Comment (jimangel) [reply content/en/docs/setup/release/notes.md]: I'm not sure, considering that directory says "These tests are importing code from kubernetes that isn't really meant to be used outside the repo." and the original link is to (https://releases.k8s.io which is the k/k github release page). It almost looks like the "examples" don't exist anymore. Thoughts @stewart-yu ?

**Comment by Bradamant3** [comment]
@chenopis can you help here? It looks to me on cursory examination as though every time we import release notes things break on this page and there's a history of manual fixes. (But I'm not completely sure -- this is a superficial conclusion.) And we've also got PR #10692 which seems to (try to) address the same issue.

/assign

File: content/en/docs/setup/release/notes.md (hunk 4 above)
Comment (MengZn) [reply content/en/docs/setup/release/notes.md]: @jimangel I have confirmed that there are no examples some versions of [ kubernetes repo](https://github.com/kubernetes/kubernetes), I suggest you can use this [kubernetes website](https://github.com/kubernetes/website/tree/release-1.12/content/en/examples) ?

**Comment by MengZn** [comment]
@Bradamant3 
We can modify this article first so that it has a title.
How to improve this problem, I will open another question.
What do you think of doing this?

**Comment by Bradamant3** [comment]
@MengZn can you fix per comment ^^, and rebase to fix merge conflicts? Thanks!

**Comment by MengZn** [comment]
@Bradamant3 yes!it will be my honor (＾▽＾)

**Comment by MengZn** [comment]
@Bradamant3 I already fix merge conflicts, and fix ugly yaml.

**Pull request closed without merging**